Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles_test.py

```diff
@@ -192,10 +160,10 @@ def test_parse_key_to_context_exclude_keys_from_list():
     "command, args, response, expected_result",
     [
         (
-            download_file_command,
-            {"object_type": "drives", "object_type_id": "123", "item_id": "232"},
-            File,
-            commands_expected_results["download_file"],
+                download_file_command,
```

Comment:
linting here looks weird, why did you add the tab here?

Reply:
I didn't. probably the reformat file and I haven't noticed. I see it elsewhere as well. I'll fix it.